server/models/remote_provider.go

```diff
@@ -310,7 +311,38 @@ func (l *RemoteProvider) InitiateLogin(w http.ResponseWriter, r *http.Request, _
 			"meshery_version":  []string{mesheryVersion},
 			"ref":              refURL,
 		}
+		if viper.GetBool("PLAYGROUND") {
+			ep, exists := l.Capabilities.GetEndpointForFeature(PersistAnonymousUser)
+			if !exists {
+				l.Log.Warn(ErrInvalidCapability("PersistAnonymousUser", l.ProviderName))
+				return
+			}
+			client := &http.Client{}
+			req, _ := http.NewRequest("GET", l.RemoteProviderURL+ep, nil)
+			req.Header.Set("X-API-Key", GlobalTokenForAnonymousResults)
```

Comment:
The remote provider should be offering this and it should not be a burden on Meshery Server to request it.

@MUzairS15 @Jougan-0, we might need to go over notes from our in-depth architecture discussion.

Reply:
I don't the question here? 
We need to make a rest call to an endpoint on remote provider to get the token and for authentication we need something so we are using the special token for authentication.
I might be wrong on the interpretation of the question.

Reply:
Let's use a sequence diagram to be of the same understanding. 

I've been hopeful to avoid a number of requests back and forth between server and remote provider.